qiskit_experiments/data_processing/data_processor.py

```diff
@@ -125,79 +190,117 @@ def _call_internal(
                 then all nodes in the data processing chain will be called.
 
         Returns:
-            datum_ and history if with_history is True or datum_ if with_history is False.
+            When ``with_history`` is ``False`` it returns an numpy array of processed data.
+            Otherwise it returns a tuple of above with a list of intermediate data at each step.
```

Comment:
what is meant by `a tuple of above`?

Reply:
Fixed in https://github.com/Qiskit/qiskit-experiments/pull/481/commits/e4914694683ac8cadb5d61ccbe36f834651f85be